docs/02.API-REFERENCE.md

```diff
@@ -3406,6 +3415,74 @@ jerry_create_string_sz (const jerry_char_t *str_p,
 - [jerry_create_string_from_utf8](#jerry_create_string_from_utf8)
 
 
+## jerry_create_regexp
+
+**Summary**
+
+Returns a jerry_value_t RegExp object or an error, if constructing the object fails.
+Optional flags can be set using [jerry_regexp_flags_t](#jerry_regexp_flags_t);
+
+**Prototype**
+```c
+jerry_value_t
+jerry_create_regexp (jerry_char_t pattern, jerry_regexp_flags_t flags);
```

Comment:
The signature isn't correctly copied from the header, see: `(const jerry_char_t *pattern`